python/tvm/api.py

```diff
@@ -300,7 +300,8 @@ def scan(init, update, state_placeholder, inputs=None, name="scan", tag=""):
     return res[0] if len(res) == 1 else res
 
 
-def extern(shape, inputs, fcompute, name="extern", dtype=None, tag=""):
+def extern(shape, inputs, fcompute, name="extern", dtype=None, in_data_alignment=-1,
+           out_data_alignment=-1, tag=""):
```

Comment:
there can be more configurations for buffer, to make this completes, we can just do in_buffers, and out_buffers